### PR TITLE
update changelogData.lua and changelog.md

### PR DESCRIPTION
**Pull request by JaggedAppliance**

### CODE REVIEW

**Comment by shalkya** [comment]
travis : luac: ./lua/ui/lobby/changelogData.lua:178: '}' expected (to close '{' at line 6) near ''       - Crash Aoe: 8 → 10''
i will look more in detail later

File: changelog.md [hunk 1]
```diff
@@ -1,17 +1,119 @@
...
+            - Build range: 5 → 6
+    - Tech 3 Land 
+        - Tanks/Bots
+            - Harbinger 
+                - Health: 3050 → 3600
+                - Range: 26 → 27
+                - Damage: 320 → 160
+                - Rate of Fire: 1 → 2
+                - Shield recharge time: 40 → 30
+            - Titan
```
Comment (shalkya) [inline changelog.md:51]: Acceleration: 4 → 4.2
from patchnote

File: changelog.md [hunk 2]
```diff
@@ -49,24 +158,51 @@ Patch 3714 (20 April, 2020)
...
+        - Aeon
+            - Energy Maintenance: 250 → 500
+        - UEF
+            - Energy Maintenance: 250 → 500
+        - Cybran
+            - Energy Maintenance: 400 → 700
+            
 - Seraphim ACU
     - regen field
         - regen ceiling:
```
Comment (shalkya) [inline changelog.md:208]: SCU: 30 hp/s
from https://content.faforever.com/patchnotes/3714.html

Comment (shalkya) [reply changelog.md:208]: SCU: 140 hp/s
for advanced regen aura

File: changelog.md (hunk 1 above)
Comment (JaggedAppliance) [reply changelog.md:51]: removed from patchnotes

File: changelog.md (hunk 2 above)
Comment (JaggedAppliance) [reply changelog.md:208]: added

**Pull request merged**